### DISCUSSION TITLE
Pull problem

### DISCUSSION THREAD

**Discussion by daryl5f** [thebaselab/codeapp#1049] [Q&A]
I ran code in China to clone a link, but the pull failed, I don't know because of my network problems in China or the bug of the software itself, I hope you can give me an answer!
![Uploading IMG_2992.png…]()

**Comment by bummoblizard** [comment]
您好，虽然我看不到截图，但这应该与网络没有关系。请问它是否私人repository? 请检查是否在设定页中输入正确的凭证，如果是GitHub，请使用个人访问令牌或SSH认证。

https://docs.github.com/zh/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens

https://code.thebaselab.com/guides/version-control
